scripts/ci/cut_ironclaw_release.py

```
@@ -166,22 +151,75 @@ def _checked_out_sha(candidate_root: Path) -> str:
-    # family move (PROPOSAL §5) would otherwise make this raise FileNotFoundError
-    # (or, worse, silently resolve nothing) once the candidate commit has moved
-    # past the flat layout (docs/reborn/target-architecture/CHECKLIST.md WS10).
+    # Resolve the shipping Cargo PACKAGE through Cargo's authoritative
+    # workspace membership. Directory names are not a stable release contract:
+    # supported release branches use `ironclaw_reborn_cli`, while main uses
+    # `app/ironclaw_cli` after the target-architecture move. A filesystem scan
+    # is insufficient because it can include manifests cargo-dist will ignore.
+    candidate_root = candidate_root.resolve()
+    try:
+        metadata_result = subprocess.run(
+            ["cargo", "metadata", "--no-deps", "--format-version", "1"],
```

Ruff (0.16.1)

[error] Starting a process with a partial executable path

(S607)